src/test/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexp/InputRegexpSemantic2.java

```diff
@@ -1,14 +1,9 @@
 /*
 Regexp
 format = This\\stext is not in the file
-message = (default)null
-illegalPattern = (default)false
-duplicateLimit = (default)0
-errorLimit = (default)100
-ignoreComments = (default)false
```

Comment:
Please keep it, we do explicit default to avoid looking at doc on what is default

Reply:
Done